packages/rxjs/useObservable/index.ts

```diff
@@ -0,0 +1,12 @@
+import { Observable } from 'rxjs'
+import { Ref, ref } from 'vue-demi'
+import { tryOnUnmounted } from '../../core/utils'
+
+export function useObservable<H>(observable: Observable<H>): Readonly<Ref<H>> {
+  const value = ref<H | undefined>()
+  const subscription = observable.subscribe(val => (value.value = val))
+  tryOnUnmounted(() => {
+    subscription.unsubscribe()
+  })
+  return value as Readonly<Ref<H>>
```

Comment:
here i wanna add readonly but vue-demi doesn't have it

Reply:
Yeah, VCA does not support `readonly` yet. We should avoid using it.

https://github.com/vuejs/composition-api#missing-apis